plugins/org.jboss.tools.runtime.reddeer/src/org/jboss/tools/runtime/reddeer/utils/FuseShellSSH.java

```diff
@@ -1,26 +1,23 @@
-package org.jboss.tools.runtime.reddeer.view;
+package org.jboss.tools.runtime.reddeer.utils;
 
 import java.io.IOException;
 
+import org.jboss.reddeer.common.logging.Logger;
 import org.jboss.reddeer.common.wait.AbstractWait;
 import org.jboss.reddeer.common.wait.TimePeriod;
 import org.jboss.reddeer.swt.impl.toolbar.DefaultToolItem;
-import org.jboss.reddeer.workbench.impl.view.WorkbenchView;
-import org.jboss.tools.runtime.reddeer.utils.ShellManager;
 
 import com.jcraft.jsch.JSchException;
 
 /**
- * Performs operations with the <i>Fuse Shell</i> in <i>Terminal</i> view. Command execution is performed via SSH (not
+ * Performs operations with the <i>Fuse Shell</i>. Command execution is performed via SSH (not
  * via JBoss Fuse Tooling).
  * 
  * @author tsedmik
  */
-public class TerminalView extends WorkbenchView {
+public class FuseShellSSH {
```

Comment:
_FuseSSHShell_ cannot be moved to _org.jboss.tools.fuse.reddeer_. It will create a cycle in dependencies

org.jboss.tools.fuse.reddeer <--> org.jboss.tools.runtime.reddeer

IMHO the mentioned class should be in runtime module. It is related to Fuse runtime.